### PR TITLE
Ruby 2.2.2

### LINKED ISSUE #37: Capistrano
Add capistrano configuration for deployment

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**
Closes #38

### CODE REVIEW

**Comment by josephbridgwaterrowe** [comment]
@michaelachrisco please review.

File: db/migrate/20150623230814_add_fields_to_customers.rb [hunk 1]
```diff
@@ -2,6 +2,6 @@ class AddFieldsToCustomers < ActiveRecord::Migration
   def change
     add_column :customers, :bill_to_location_id, :integer
     add_column :customers, :ship_to_location_id, :integer
-    add_column :customers, :is_tax_excempt, :integer
+    add_column :customers, :is_tax_exempt, :integer
```
Comment (michaelachrisco) [inline db/migrate/20150623230814_add_fields_to_customers.rb]: Snuck this in :wink:. Ill let it slide.

Comment (michaelachrisco) [inline db/migrate/20150623230814_add_fields_to_customers.rb]: Although you need to run the migration for your schema.rb to pick up the changes.

Comment (josephbridgwaterrowe) [inline db/migrate/20150623230814_add_fields_to_customers.rb]: Will do, thanks :smile:

**Comment by michaelachrisco** [comment]
If/when this passes, rebase and I will merge. Thanks.

**Comment by josephbridgwaterrowe** [comment]
:+1:

**Comment by josephbridgwaterrowe** [comment]
Thanks!

**Pull request merged**